jac-uk/apply
# Feature/1089 Transfer character checks to PSD

## What's included?
Closes #1089 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

Use the following email addresses (password: 1qazZAQ!1qaz) to sign in to the apply side:
- application-0001@jac-dummy-email.jac
- application-0002@jac-dummy-email.jac
- application-0003@jac-dummy-email.jac
- application-0004@jac-dummy-email.jac
- application-0005@jac-dummy-email.jac
- application-0006@jac-dummy-email.jac
- application-0007@jac-dummy-email.jac
- application-0008@jac-dummy-email.jac

Steps:
1. Go to "Current applications" and click the button "Pre Selection Day Questionnaire" under exercise JAC00685.
2. Check if you can see the "Character Checks" in the list.
3. Go to "Character Checks" and check if you can complete the character checks.
4. Click "Save and continue" and check if "Character Checks" is marked as "DONE".

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Demo:

https://github.com/jac-uk/apply/assets/79906532/14e1f214-e6c7-46e1-9c40-b83a2acad5bf

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
